packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -100,6 +119,26 @@ export const CloudPulseResourcesSelect = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [resources, region, xFilter, resourceType]);
 
+    // selected resources will appear at the top in the autcomplete popper
+    const sortedResourcesList = React.useMemo<CloudPulseResources[]>(() => {
+      return [...getResourcesList].sort((resource_a, resource_b) => {
+        const aIsSelected = selectedResources?.some(
+          (item) => item.label === resource_a.label
+        );
+        const bIsSelected = selectedResources?.some(
+          (item) => item.label === resource_b.label
+        );
+
+        if (aIsSelected && !bIsSelected) {
+          return -1;
+        }
+        if (!aIsSelected && bIsSelected) {
+          return 1;
+        }
+        return 0;
+      });
+    }, [getResourcesList, selectedResources]);
```

Comment:
Please check if there any optimisation can be done on this method, if not it is okay